geomstats/geometry/special_euclidean.py

```diff
@@ -772,3 +705,32 @@ def _exponential_matrix(self, rot_vec):
         exponential_mat = term_1 + term_2
 
         return exponential_mat
+
+
+class SpecialEuclidean(_SpecialEuclidean3Vector, _SpecialEuclideanMatrix):
+    r"""Class for the special euclidean groups.
+
+    Parameters
+    ----------
+    n : int
+        Integer representing the shapes of the matrices : n x n.
+    point_type : str, {\'vector\', \'matrix\'}
+        Representation of the elements of the group.
+    epsilon : float, optional
+        precision to use for calculations involving potential divison by 0 in
+        rotations
+        default: 0
+    """
+
+    def __new__(cls, n, point_type='matrix', epsilon=0.):
+        """Instantiate a special euclidean group.
+
+        Select the object to instantiate depending on the point_type.
+        """
+        if n == 3 and point_type == 'vector':
+            return _SpecialEuclidean3Vector(epsilon)
+        if n != 3 and point_type == 'vector':
+            raise NotImplementedError(
+                'SE(n) is only implemented in matrix representation'
```

Comment:
maybe clearer the other way around: SE(n) in only implemented in vector representation for n=3